docs/terminal/atma.md

```diff
@@ -64,24 +64,56 @@ ATMA should note that all prop-preferential SIDs have a hold-down altitude restr
 
 Aircraft under the jurisdiction of ATMA may be track-shortened or assigned a heading to assist with sequencing or for separation. 
 
-ATMA may track-shorten where practical within their sector. ATMA may track-shorten aircraft direct to `KARRL` without coordination, provided that it their flight plan has been amended.
+If a SID is cancelled or the aircraft is given a shortcut ATMA shall note in the tag where the departing aircraft is tracking. 
 
-As track-shortening direct to `TAPAT` or `BOGUN` can interfere with arrivals within OCR, these shall be co-ordinated.
+ATMA shall instruct any aircraft on a radar heading to `report heading` to the next sector. 
 
-### Short-Sector STARs
+!!!!
+
+#### OCR 
+
+ATMA shall manually hand departing international flights to OCR as soon as practicable ensuring that aircraft are; on the SID, tracking to the SID endpoint, thence SID transition or on a vector and will remain clear of all aircraft under control. **Exceptions to the above are:**
+
+- Aircraft to PEBLU, VELMO, GULUT, DUGAN, ELNOS may be tracked direct to these points by TMA
+- Aircraft to MADEP/UPLAR may be tracked to BOGUN/TAPAT respectivly
+
+#### RAN
```

Comment:
I don't believe the "Flights from" column is necessary, as all of them are ATMA. If there were one or two different ADEPs, then it would make sense. I would also suggest swapping the "may be cleared" and "provided that" columns around as it reads better. IE - qualification then action.

Also, please standardize the amount of full-stops... should be an ellipsis which is three.